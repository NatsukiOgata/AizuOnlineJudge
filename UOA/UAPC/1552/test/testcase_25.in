1000 7 4 0 18 4 2 3 7 10 3 19 19 2 8 13 17 9 18 6 9 9 13 4 12 4 6 19 17 16 0 2 5 15 16 11 14 16 1 1 17 0 4 0 0 16 6 3 2 13 19 10 18 10 11 13 16 15 19 14 18 4 10 11 16 9 15 7 11 17 15 13 0 3 10 5 1 17 18 6 8 16 1 12 2 5 16 1 0 8 9 17 14 13 3 19 8 12 7 0 9 3 0 12 12 2 10 6 7 15 9 17 12 9 12 6 2 13 12 15 2 18 15 18 17 13 16 10 15 3 15 8 15 9 15 1 0 18 9 18 18 13 11 4 1 16 17 15 13 8 6 2 8 11 18 18 8 1 15 14 0 8 13 0 18 19 14 1 1 7 0 6 0 19 16 1 17 4 4 13 18 2 0 7 0 12 17 3 2 6 19 2 19 7 9 5 18 19 12 3 19 2 11 3 17 18 16 6 4 15 5 8 0 2 0 1 1 8 11 14 2 8 12 1 8 7 18 1 8 8 14 6 4 5 9 19 15 5 11 13 15 8 7 15 8 4 9 18 2 9 12 9 4 0 13 12 0 5 15 12 10 3 14 9 13 15 4 15 5 19 2 17 0 5 10 11 7 2 5 5 15 18 11 0 15 6 6 10 4 18 3 8 5 10 11 12 10 8 12 8 15 15 5 7 12 3 13 7 11 2 13 7 9 8 9 13 6 13 1 6 14 1 12 0 9 17 0 10 1 17 13 14 13 0 0 12 19 17 13 14 4 15 5 2 9 6 3 8 17 8 8 14 10 2 18 8 15 7 16 7 4 3 18 11 0 7 4 1 3 1 12 3 12 12 14 10 5 17 7 9 12 10 19 19 18 18 11 9 14 4 5 0 3 0 13 2 15 13 19 7 18 9 6 2 17 15 6 9 6 0 0 7 4 10 3 4 13 14 6 17 5 6 14 0 18 1 9 14 0 16 19 9 2 1 1 15 8 18 8 1 8 15 16 0 3 9 2 17 18 6 8 11 5 10 4 5 8 12 10 5 4 2 15 1 14 4 5 14 13 14 10 17 2 12 2 14 7 13 14 13 4 16 4 10 13 4 12 5 3 16 11 6 11 0 2 7 19 10 1 15 15 15 11 13 4 14 16 15 7 11 12 18 2 16 10 7 9 0 4 13 13 9 19 9 6 16 5 12 14 16 15 7 1 0 14 19 6 8 14 13 5 13 6 7 10 14 3 0 12 0 0 4 13 1 1 19 8 17 4 8 7 3 6 19 14 7 8 5 10 14 13 9 0 18 8 7 0 2 17 10 14 2 6 16 16 4 0 16 3 15 13 3 1 18 11 12 9 3 12 5 16 14 0 18 0 1 5 3 4 7 14 8 5 9 7 6 18 11 19 15 11 5 13 9 14 8 9 0 14 0 13 11 15 5 13 4 10 6 19 19 4 11 6 19 18 11 14 13 7 18 17 6 14 0 16 11 19 7 12 6 15 12 19 0 2 0 2 18 18 9 18 16 15 14 3 10 8 18 3 3 13 3 2 10 15 12 15 5 5 8 19 14 18 10 11 19 3 15 15 16 15 3 13 18 11 7 18 11 8 12 16 4 0 10 18 13 12 12 9 14 4 17 5 18 18 1 8 14 8 10 16 14 19 3 18 18 12 18 5 17 15 12 1 0 0 5 15 7 15 12 7 17 0 9 16 2 14 6 0 9 10 0 11 6 0 3 2 10 0 18 16 18 6 18 19 15 5 13 11 8 6 1 2 2 9 12 15 0 16 0 11 14 12 18 6 11 1 11 16 4 5 14 5 5 18 19 3 18 11 0 11 12 0 15 16 1 12 9 4 9 3 6 13 9 11 10 16 6 16 1 8 7 9 16 10 19 17 2 6 0 14 0 17 3 8 1 6 7 1 15 14 7 16 1 1 7 18 15 13 11 16 8 0 16 3 7 7 2 7 3 12 13 0 9 11 17 9 1 1 17 15 17 5 12 1 16 6 19 9 0 18 12 5 0 7 3 10 2 3 18 12 4 17 12 18 9 17 13 0 4 5 3 4 4 17 9 17 2 11 0 18 0 7 16 19 0 16 10 0 15 2 12 15 13 7 0 8 18 6 4 2 10 10 1 8 13 19 8 8 9 4 7 12 7 11 16 19 17 7 0 12 9 4 0 6 15 19 2 17 0 14 0 8 17 7 18 14 12 3 6 8 11 19 5 10 18 6 12 15 12 15 3 19 3 11 5 12 5 8 13 5
