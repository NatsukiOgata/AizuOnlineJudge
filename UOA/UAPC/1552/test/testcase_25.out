15 18 10 318 315
